model/daal4py/examples/lr/lr.py

```diff
@@ -12,7 +13,10 @@
 train(model, "train.csv")
 
 # Assess accuracy (alternate way of specifying data source)
-print("Accuracy:", accuracy(model, CSVSource(filename="test.csv")))
+mse_accuracy = MeanSquaredErrorAccuracy()
+print(
+    "Accuracy:", accuracy(model, mse_accuracy, CSVSource(filename="test.csv"))
+)
```

Comment:
Since we have the `model` argument named `model`, let's name the accuracy scorer `scorer`, for consistency.

```suggestion
scorer = MeanSquaredErrorAccuracy()
print(
    "Accuracy:", accuracy(model, scorer, CSVSource(filename="test.csv"))
)
```